ipaserver/install/server/replicainstall.py

```diff
@@ -836,14 +836,12 @@ def install_check(installer):
             root_logger.debug('No IPA DNS servers, '
                               'skipping forward/reverse resolution check')
 
-        kra_enabled = remote_api.Command.kra_is_enabled()['result']
```

Comment:
No, you don't understand it. KRA would not be installed. See `kra.py:install()` for details.